client/components/Map.jsx

```diff
@@ -20,28 +22,31 @@ const mapStateToProps = ({map: {viewport, pinLocations}}) => ({
 const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
 
 const Map = (props) => {
+    useEffect(() => {
+        props.getCoordinates();
 
-  // 
-
-    console.log(props.pinLocations)
+    }, [])
 
+    // console.log('pinLocations are', props.pinLocations)
     return (
 
       <ReactMapGL mapboxApiAccessToken = {process.env.REACT_APP_MAPBOX_ACCESS_TOKEN} mapStyle='mapbox://styles/chloelu29/ckxwc6fdl24xz14phmw34jij0' {...props.viewport} onViewportChange={(newViewport) => {
           // console.log(props.viewport)
           props.setMap(newViewport)
       }}>
           {props.pinLocations.map((el, key) => {
-              
-              return (
-              <Marker key={key + 1} latitude={el.latitude} longitude={el.longitude}>
-              {/* button onclick post pops up */}
-                  <button onClick={(e) => {
-                      props.changeActivePost(el.latitude, el.longitude)
-
-                  }}style={{backgroundColor: 'transparent'}}><img src={'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRCkP7QFhZEIqrhTgIxZIJKmW0zBB50L1Fc6xVI4T7U8cNaNHxEOjxu3Wl578bNT6DHmsY&usqp=CAU'} alt='pin' style={{backgroundColor: 'transparent', height: '50px', width: '50px'}}/></button>
-              </Marker>
-              )
+              console.log('el is', el)
+            return (
+            <Marker key={key + 1} latitude={el.latitude} longitude={el.longitude} address={el.address} id={el.id}>
+            {/* button onclick post pops up */}
+                <button onClick={(e) => {
+
+                // console.log(e.target.value)
+                props.changeActivePost(el.id)
+
+                }}style={{backgroundColor: 'transparent'}}><img src={'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRCkP7QFhZEIqrhTgIxZIJKmW0zBB50L1Fc6xVI4T7U8cNaNHxEOjxu3Wl578bNT6DHmsY&usqp=CAU'} alt='pin' style={{backgroundColor: 'transparent', height: '50px', width: '50px'}}/></button>
+            </Marker>
+            )
           }
           )}
           <Marker key={0} latitude={40.7128} longitude={-74}>
```

Comment:
Will we still need this Marker that has a set latitude and longitude, or was it just for testing?

Reply:
just for testing. that's for the original pin. i'll delete that.